index.js

```diff
@@ -1,18 +1,151 @@
 const readline = require('readline');
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// Valid string inputs ---------------------------------------------
+const correctAnswer = ['yes', 'y', 'ys'];
+const wrongAnswer = ['no', 'n'];
+const highAnswer = ['higher', 'h', 'high'];
+const lowAnswer = ['lower', 'l', 'low'];
+const playComputerGuessing = ['computer', 'comp', 'c'];
+const playPlayerGuessing = ['player', 'play', 'p'];
+const validGameTypeInput = [...playComputerGuessing, ...playPlayerGuessing];
+let setRangeMax = process.argv[3] || 100;
+let setRangeMin = process.argv[2] || 1;
```

Comment:
Good use of argv here, maybe you could write a check that switches the max and min if the user were to input the max and min in the wrong order. Something like:
```js
// max = 1, min = 100
if ( setRangeMax < setRangeMin ) {
  let temp = setRangeMax;  // temp = 1
  setRangeMax = setRangeMin;  // max = 100
  setRangeMin = temp;  // min = 1
}
```
Another method that is a cool feature of javascript to switch the values of variables is:
```js
if ( setRangeMax < setRangeMin ) {
  [ setRangeMin, setRangeMax ] = [ setRangeMax, setRangeMin ]
}
```
This second method is something called destructuring assignment, which is a feature of JS since ES6 ( which is just the version of javascript released in 2015 ). [Here](https://medium.com/better-programming/how-swap-two-values-without-temporary-variables-using-javascript-8bb28f96b5f6) is a link to an article about swapping variables and [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) is a link to the MDN page on destructuring assignment.